src/sstdt/bin/main.rs

```diff
@@ -7,24 +7,24 @@ mod startin;
 #[macro_use]
 extern crate log; //info/debug/error
 
-use std::fs::File;
```

Comment:
this line needs to stay if we want to use the input file below (super useful to debug!(